lib/kafka_ex/server.ex

```diff
@@ -73,6 +75,17 @@ defmodule KafkaEx.Server do
     {:ok, state, timeout | :hibernate} |
     :ignore |
     {:stop, reason :: any} when state: any
+  @callback kafka_server_connect(state :: State.t) ::
+    {:noreply, new_state} |
+    {:noreply, new_state, timeout | :hibernate} |
+    {:stop, reason :: term, new_state} when new_state: term
+  @callback kafka_server_ready_check(state :: State.t) ::
```

Comment:
Maybe be worth adding in space between each callback for readability

Reply:
this is just following the style convention set already.

Reply:
:+1: